bedrock/firefox/tests/test_base.py

```diff
@@ -542,6 +542,54 @@ def test_fx_99_0_0_en(self, render_mock):
 
     # end 99.0 whatsnew tests
 
+    # begin 101.0 whatsnew tests
+
+    def test_fx_101_0_0_en(self, render_mock):
+        """Should use whatsnew-fx101-vpn-en template for 101.0 in English"""
```

Comment:
Suggestion (non-blocking): is it worth adding some tests for the experiment variation templates?